.github/workflows/build_windows_release_candidate.yml

```
@@ -0,0 +1,32 @@
+name: Build Windows Release Candidate
```

Perhaps we make it clearer that it's not signed:
```suggestion
name: Build Windows Unsigned Release Candidate
```